src/javascript_bindings.pyx

```diff
@@ -100,10 +100,14 @@ cdef class JavascriptBindings:
             properties = self.properties
             objects = {}
             for objectName in self.objects:
-                methods = {}
-                for methodName in self.objects[objectName]:
-                    methods[methodName] = None
-                objects[objectName] = methods
+                attrs = {}
+                for name, value in self.objects[objectName].items():
+                    if self.IsValueAllowedRecursively(value) and \
+                            name == '__call__' or not name.startswith('_'):
```

Comment:
Another one to the list:
6. You have changed SetObject() body to use inspect.getmembers(predicate=False) when allow_properties=True. This changes behavior significantly and might return lots of values. Can you please print the `attrs` variable to make sure there are only values that are supposed to be there?